index.js

```diff
@@ -16,6 +16,8 @@ module.exports = {
 
    'extends': 'eslint:recommended',
 
+   'ignorePatterns': [ '!.vuepress' ],
```

Comment:
Could we include a brief comment about why we are using a negated pattern inside of `ignorePatterns`?

Reply:
As a reference for that, the docs on eslint's [ignoring code](https://eslint.org/docs/latest/user-guide/configuring/ignoring-code#the-eslintignore-file) page discusses how node_modules, dot-files, and dot-folders are all implicitly ignored.